src/main/java/com/iexec/worker/config/WorkerConfigurationService.java

```diff
@@ -48,8 +50,20 @@ public String getWorkerWalletAddress() {
         return credentialsService.getCredentials().getAddress();
     }
 
-    public String getResultBaseDir() {
-        return resultBaseDir + File.separator + workerName;
+    public String getWorkerBaseDir() {
+        return workerBaseDir + File.separator + workerName;
+    }
+
+    public String getTaskBaseDir(String chainTaskId) {
+        return getWorkerBaseDir() + File.separator + chainTaskId;
+    }
+
+    public String getTaskInputDir(String chainTaskId) {
+        return getWorkerBaseDir() + File.separator + chainTaskId + FileHelper.SLASH_INPUT;
+    }
+
+    public String getTaskOutputDir(String chainTaskId) {
```

Comment:
Are we sure about that?

Reply:
It's the location of the output directory that would be zipped after running computations (which contains iexec_out and stdout.txt )